conferences/forms.py

```diff
@@ -50,3 +50,8 @@ def _pays_for(d):
 
         if len(errs) > 0:
             raise forms.ValidationError(errs)
+
+    def disable(self):
+        for field in self.fields:
+            if field not in ['contact', 'shirt_size', 'shirt_type']:
+                self.fields[field].disabled = True
```

Comment:
a czy to musisz customowo obsługiwać w szablonie? Możesz też zrobić w ten sposób: `self.fields[field].widget.attrs['disabled'] = True`